tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,38 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        product = entities.Product(organization=self.org).create()
+        repos_amount = randint(1, 5)
+        repos = [
+            _create_repository(product, name=gen_string('alpha'))
+            for _
+            in range(repos_amount)
+        ]
+        self.assertEqual(len(product.read().repository), repos_amount)
+
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.org,
+        ).create()
+        content_view.repository = repos
+        content_view = content_view.update(['repository'])
+
+        self.assertEqual(len(content_view.repository), repos_amount)
+
+        content_view.repository = [
+            repo.read() for repo in content_view.repository
+        ]
+
+        for attr in ('id', 'content_type', 'docker_upstream_name'):
+            self.assertEqual(
+                {getattr(repo, attr) for repo in repos},
+                {getattr(repo, attr) for repo in content_view.repository},
+            )
+
+        for repo in content_view.repository:
+            self.assertEqual(repo.content_type, u'docker')
+            self.assertEqual(repo.docker_upstream_name, u'busybox')
```

Comment:
The previous loop already asserts that each repo has a content-type of 'docker' and an upstream name of 'busybox'. If you want to be completely sure, then drop some of the code above:

```python
        self.assertEqual(
            {repo.id for repo in repos},
            {repo.id for repo in content_view.repository},
        )
        for repo in content_view.repository:
            self.assertEqual(repo.content_type, u'docker')
            self.assertEqual(repo.docker_upstream_name, u'busybox')
```